web/src/app/admin/AdminConfirmDialog.js

```diff
@@ -70,9 +70,9 @@ export default class AdminConfirmDialog extends React.PureComponent {
           })),
         )}
         form={
-          <List>
+          <List data-cy='confirmation-diff'>
             {changes.map(c => (
-              <ListItem divider key={c.id}>
+              <ListItem divider key={c.id} data-cy={c.id}>
```

Comment:
Since we search for `data-cy` attrs globally, we should qualify this one, maybe `data-cy={'confirmation-diff-'+c.id}` or `'diff-'+c.id`, just so that there's never a possible conflict.